js/renovation/ui/grids/data_grid_light/data_grid_light.tsx

```diff
@@ -61,11 +83,13 @@ export const viewFunction = (viewModel: DataGridLight): JSX.Element => (
     width={viewModel.props.width}
     {...viewModel.restAttributes} // eslint-disable-line react/jsx-props-no-spreading
   >
-    <ValueSetter type={AllItems} value={viewModel.props.dataSource} />
+    <ValueSetter type={LocalData} value={viewModel.localData} />
```

Comment:
Why do we need to use the "Local" prefix? Are we going to add "Remote" data? This value will contain an array of items. So let it be DataItems or something like that.